frontend/src/app/base-enum-pipe.ts

```diff
@@ -0,0 +1,15 @@
+import { PipeTransform } from '@angular/core';
+
+export class BaseEnumPipe implements PipeTransform {
+
+  constructor(private translations: Array<{key: string; translation: string}>) { }
```

Comment:
That's what I began with, but the order of keys/entries in an object is undefined, and we want to preserve order of enums generally (the "other" gender, for example, would look weird if it appeared first).